runtime/pack/dist/opt/termdebug/plugin/termdebug.vim

```diff
@@ -50,6 +50,13 @@ if exists('g:termdebug_loaded')
     finish
 endif
 g:termdebug_loaded = true
+g:termdebug_is_running = false
```

Comment:
do we still need `running` var, now that we have the `g:termdebug_is_running` var?

Reply:
I think it is still needed since it relates to the code being debugged, not termdebug itself. Look for example to `HandleCursors()` function.

Reply:
alright